data/itsshivamsrivastava.json

```diff
@@ -0,0 +1,48 @@
+{
+  "name": "Shivam Srivastava",
+  "type": "personal",
+  "bio": "Fresher | Aspiring Web Developer | Seeking Entry- level Programming Position",
+  "links": [
+    {
+      "name": "Follow me on Instagram",
+      "url": "https://www.instagram.com/itsshivamsrivastava/",
+      "icon": "FaInstagram"
+    },
+    {
+      "name": "Follow me on LinkedIn",
+      "url": "https://www.linkedin.com/in/itsshivamsrivastava/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/itsshivamsrivastava",
+      "icon": "FaGithub"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/shivam_142",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Facebook",
+      "url": "https://www.facebook.com/itsmeshivamsrivastava",
+      "icon": "FaFacebook"
+    }
+  ],
+  "socials": [
+  { "icon": "FaInstagram", "url": "https://www.instagram.com/itsshivamsrivastava/" },
+  { "icon": "FaGithub", "url": "https://github.com/itsshivamsrivastava" },
+  { "icon": "FaLinkedin", "url": "https://www.linkedin.com/in/itsshivamsrivastava/" },
+  { "icon": "FaFacebook", "url": "https://www.facebook.com/itsmeshivamsrivastava/" },
+  { "icon": "FaTwitter", "url": "https://twitter.com/shivam_142/" },
+  { "icon": "FaYoutube", "url": "https://www.youtube.com/channel/UCxLg2xlwOpmRTjeKfgayt8w" }
+  ],
+  "tags": [ 
+  "Open Source", "Javascript", "ReactJS", "NodeJS", "DevOps", "DevRel", "TypeScript", "NextJS" 
+  ],
+  "testimonials": [
```

Comment:
It seems like you've not yet received testimonials from these people. So it's better to remove these from your file. 🛠️

Learn more about testimonials [here.](https://linkfree.io/docs/how-to-guides/testimonials) 🙂